js/utils/create-out-add.js

```diff
@@ -0,0 +1,41 @@
+import { getRandomNumber } from './get-random-number.js';
+import { getFloatingPointNumber } from './get-random-floating-point.js';
+import { getNumberAvatar } from './get-number-avatar.js';
+import { getRandomFeatures } from './get-random-features.js';
+import { getRandomPhotos } from './get-random-photos.js';
+import { TITLES, TYPES, TIMES, DESCRIPTIONS, minLat, maxLat, minLng, maxLng } from '../data.js';
+
+// out add
+
+const createOutAdd = () => {
+  const location = {
+    'lat': getFloatingPointNumber(minLat, maxLat, 5),
+    'lng': getFloatingPointNumber(minLng, maxLng, 5),
```

Comment:
```suggestion
    lat: getFloatingPointNumber(minLat, maxLat, 5),
    lng: getFloatingPointNumber(minLng, maxLng, 5),
```
строки просто так не добавляй
(исправь везде 🙏)